xla/debug_options_flags.cc

```diff
@@ -877,6 +880,11 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
                 bool_setter_for(&DebugOptions::set_xla_cpu_use_mkl_dnn),
                 debug_options->xla_cpu_use_mkl_dnn(),
                 "Generate calls to MKL-DNN in the CPU backend."));
+  flag_list->push_back(
+      tsl::Flag("xla_cpu_use_mkl_dnn_with_acl",
+                bool_setter_for(&DebugOptions::set_xla_cpu_use_mkl_dnn_with_acl),
+                debug_options->xla_cpu_use_mkl_dnn_with_acl(),
+                "Generate calls to MKL-DNN using ACL (Arm Compute Library) in the CPU backend."));
```

Comment:
I don't think we need a `DebugOptions` flag for this. We just need to check whether the kernel is running on an aarch64 CPU. Please see my other comment on onednn_matmul.cc.

Reply:
Hi @penpornk, the extra reorder is required only for ACL backend. So, it would be more appropriate to check for ACL backend instead of generic aarch64 platform check.

Reply:
`__xla_cpu_runtime_oneDNNMatMul` is only called from oneDNN integration code, and we always use ACL for oneDNN aarch64 backend. So I think it's fine to just detect aarch64 CPU.